java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -602,6 +605,199 @@ public static CompletableFuture<Object> arrinsert(
                         .toArray());
     }
 
+    /**
+     * Searches for the first occurrence of a <code>scalar</code> JSON value in the arrays at the
+     * path.
+     *
+     * @param client The client to execute the command.
+     * @param key The key of the JSON document.
+     * @param path The path within the JSON document.
+     * @param scalar The scalar value to search for; JSON scalar refers to values that are not objects
+     *     or arrays.
+     * @return An array of integers. Each value is the index of the matching element in the array at
+     *     the path. The value is -1 if not found. If there are no values in the array, the return
+     *     value will be <code>null</code>
```

Comment:
Unfortunately, command return type differs depending on the path type. Please update according to #2528 (see other java commands for example, e.g. `arrtrim`)

Reply:
Applicable to all new methods